tools/make-frozen.py

```diff
@@ -50,7 +50,7 @@ def module_name(f):
 for f, st in modules:
     print("%d," % st.st_size)
 
-print("};")
+print("0};")
```

Comment:
Is this supposed to be `\\0`?

Reply:
No this is for the `uint32_t mp_frozen_str_sizes` array

Reply:
Ah, I see it now.